### PR TITLE
Feature: running first migration with alembic

### PR DESCRIPTION
**Pull request by eapg**
* Adding alembic toolkit for manage migrations with sqlalchemy
* Runnig firts migration to create db tables and constrains
* Adding Alembic and sqlalchemy versions to requirements.txt
* deleting comments from db_script
* Adding alembic commands to readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,3 +68,17 @@ Commands:
...
+### Migrations
+
+This project is Using [Alembic](https://alembic.sqlalchemy.org/en/latest/) for migrations. To
+install alembic in your project just run `pip install alembic`
+
+Commands to use alembic:
+
+* Initialize: `alembic init alembic`
+* Migration Script: `alembic revision -m "migration scrip name"`
+* Update: `alembic updgrate <target-revision>`
```
Comment (reynoldmorel) [inline README.md:81]: Is this typo `updgrate`?

File: README.md [hunk 2]
```diff
@@ -68,3 +68,17 @@ Commands:
...
+
+### Migrations
+
+This project is Using [Alembic](https://alembic.sqlalchemy.org/en/latest/) for migrations. To
+install alembic in your project just run `pip install alembic`
+
+Commands to use alembic:
+
+* Initialize: `alembic init alembic`
+* Migration Script: `alembic revision -m "migration scrip name"`
```
Comment (reynoldmorel) [inline README.md:80]: What is this for?

File: src/migrations/versions/c5d3236b81a3_db_script.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'c5d3236b81a3'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+engine = sa.create_engine('postgresql://postgres:1234@localhost/ea_restaurant')
```
Comment (reynoldmorel) [inline src/migrations/versions/c5d3236b81a3_db_script.py:18]: Why do you need this here? If you already add your db url above?

File: README.md (hunk 1 above)
Comment (eapg) [reply README.md:81]: writting error. fixed

File: README.md (hunk 2 above)
Comment (eapg) [reply README.md:80]: this command generate an empty version script just with version id, updgrade and downgrade method

File: src/migrations/versions/c5d3236b81a3_db_script.py (hunk 3 above)
Comment (eapg) [reply src/migrations/versions/c5d3236b81a3_db_script.py:18]: done.

File: README.md (hunk 2 above)
Comment (reynoldmorel) [reply README.md:80]: Could document it better? It is not clear to know that

**Review by reynoldmorel — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (eapg) [reply README.md:80]: DONE,

**Pull request merged**